luigi.cfg

```diff
@@ -0,0 +1,9 @@
+[RegionsDataset]
```

Comment:
We could add more directly to the config instead of defaults in parameters. On the other hand, this doesn't look that bad.

However, we could remove other things from the `config.yaml` and make them explicitly assigned to their task (which would be better IMHO).